WTWSMS/history/provinces/100 - Avranches.txt

```diff
@@ -22,8 +22,6 @@ religion = celtic_pagan
 terrain = forest
 # History
 1.1.1 = { # Viking takeover
-	remove_settlement = b_caen
-	remove_settlement = b_bayeux
```

Comment:
It's an obvious error, we should just change the date and let these two lines stay :)